internal/repository/mapper/userEntityMapper.go

```diff
@@ -2,8 +2,8 @@ package mapper
 
 import (
 	"github.com/google/uuid"
-	"github.com/murilogilfelpeto/ps-tag-onboarding-go/repository/entities"
-	"github.com/murilogilfelpeto/ps-tag-onboarding-go/service/models"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/internal/repository/entities"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/internal/service/models"
 )
 
 func UserToUserEntity(user models.User) entities.UserEntity {
```

Comment:
These functions need unit tests written for them since the only methods that call these functions do not have associated unit tests.

Reply:
Done